changelog.json

```
@@ -1,5 +1,22 @@
 {
   "releases": {
+    "1.6.0": [
+      "[New] Help users add their first repo during onboarding - #6474",
+      "[New] \"No local changes\" view makes it easy to find and accomplish common actions - #6445",
+      "[Added] Include ability for users to add new repositories when there are none available - #5566 #6474",
+      "[Added] Support JetBrains Webstorm as an external editor - #6077. Thanks @KennethSweezy!",
+      "[Added] Add Visual Basic syntax highlighting - #6461. Thanks @SimpleBinary!",
+      "[Fixed] Automatically locate a missing repository when it cannot be found - #6228. Thanks @msftrncs!",
+      "[Fixed] Don't include untracked files in merge commit - #6411",
+      "[Fixed] Don't show \"Still Conflicted Warning\" when all conflicts are resolved - #6451",
+      "[Fixed] Enter when using keyboard to navigate app menu executed menu action twice - #5344",
+      "[Fixed] Autocomplete selection does not overflow text area - #6459",
```

Tentative, feel free to decline: `selection` feels a bit vague.

```suggestion
      "[Fixed] Autocomplete list does not overflow commit description text - #6459",
```